### PR TITLE
Update data-storage-privacy.md

### PR DESCRIPTION
**Pull request by msbemba**
Added details about advanced hunting data

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4540

File: microsoft-365/security/defender-endpoint/data-storage-privacy.md [hunk 1]
```diff
@@ -90,6 +90,9 @@ You can choose the data retention policy for your data. This determines how long
 **At contract termination or expiration**<br>
 Your data will be kept and will be available to you while the license is under grace period or suspended mode. At the end of this period, that data will be erased from Microsoft’s systems to make it unrecoverable, no later than 180 days from contract termination or expiration.
 
+**Advanced Hunting data**<br.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/data-storage-privacy.md]:
```suggestion
**Advanced Hunting data**<br>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**